TESTING_CONTRIBUTIONS_DOCUMENTATION.md

```diff
@@ -0,0 +1,20 @@
+In order to test documentation updates prior to submitting a PR, it is recommended that you perform a local rendering of the documentation. Once rendered, view the newly-created HTML files to ensure proper presence, location and formatting of new documentation-content.
```

Comment:
This should be a section in CONTRIBUTING.md. Also, opening the PR will do generate a preview link where you can see the rendered docs.

Reply:
> This should be a section in CONTRIBUTING.md.

Ok, I'll move it.

> Also, opening the PR will do generate a preview link where you can see the rendered docs.

I'd rather have the option to _not_ have to rely on having already opened a PR – not even as `WIP` – just to view rendered documentation.